rest-api/src/main/resources/paths/studies/v5_studies_studyId_adherence_stats.yml

```diff
@@ -25,3 +25,7 @@ get:
             $ref: ../../responses/401.yml
         403:
             $ref: ../../responses/403_not_developer_researcher_study_designer_study_coordinator.yml
+        404:
+            description: The study does not have a defined schedule.
+            schema:
+                $ref: ../../definitions/message.yml
```

Comment:
Some of these are replaced with a reference to ../../responses/404_no_schedule.yml. Do you want to keep this independent for the description difference? I'll note where I find these in case you want to change them.